src/samtools/samtools_fastq/script.sh

```diff
@@ -11,7 +11,7 @@ set -e
 [[ "$par_copy_tags" == "false" ]] && unset par_copy_tags
 [[ "$par_casava" == "false" ]] && unset par_casava
 
-samtools fastq \
+samtools "$par_command" \
```

Comment:
This is one alternative approach

```suggestion
if [[ "$meta_name" == "samtools_fasta" ]]; then
  subcommand=fasta
elif [[ "$meta_name" == "samtools_fastq" ]]; then
  subcommand=fastq
else
  echo "Unrecognized component name" && exit 1
fi
samtools "$subcommand" \
```

Reply:
Yes that's much better, thanks :)